pkg/minikube/download/preload.go

```diff
@@ -53,6 +54,10 @@ var (
 	preloadStates = make(map[string]map[string]bool)
 )
 
+var (
+	windowsServerIsoURL = "https://go.microsoft.com/fwlink/p/?LinkID=2195280&clcid=0x409&culture=en-us&country=US"
+)
+
```

Comment:
This should be passed in from the Cluster Config, not hardcoded here

Reply:
I am having problems having this in Cluster Config. In the latest change I have placed this in a const